core/chains/legacyevm/chain.go

```diff
@@ -28,6 +29,9 @@ import (
 	"github.com/smartcontractkit/chainlink-evm/pkg/monitor"
 	evmtypes "github.com/smartcontractkit/chainlink-evm/pkg/types"
 	ubig "github.com/smartcontractkit/chainlink-evm/pkg/utils/big"
+	tronkeystore "github.com/smartcontractkit/chainlink-tron/relayer/keystore"
+	tronclient "github.com/smartcontractkit/chainlink-tron/relayer/sdk"
+	trontxm "github.com/smartcontractkit/chainlink-tron/relayer/txm"
```

Comment:
Importing tron from here cannot be allowed. Both because core must not import tron, but also this is moving to chainlink-evm soon.